### DISCUSSION TITLE
Cannot implement an alternative WebUI

### DISCUSSION THREAD

**Discussion by thibaudbrg** [haugene/docker-transmission-openvpn#2217] [Q&A]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Docker run config used

  transmission-vpn:
    image: haugene/transmission-openvpn:latest
    container_name: transmission-vpn
    restart: unless-stopped
    networks:
      t2_proxy:
        ipv4_address: 192.168.90.169
    # ports:
    #  - "$TRANSMISSION_PORT:9091"
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun
    # dns:
    #   - 1.1.1.1
    #   - 1.0.0.1
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - $DOCKERDIR/appdata/transmission-vpn/data:/data
      - $DOCKERDIR/appdata/transmission-vpn/config:/config
      - $DOCKERDIR/appdata/transmission-vpn/:/etc/openvpn/cyberghost/
      - $DATADIR/torrents:/data/torrents
    environment:
      <<: *default-tz-puid-pgid
      OPENVPN_PROVIDER=cyberghost
      OPENVPN_USERNAME=**None**
      OPENVPN_PASSWORD=**None**
      OPENVPN_CONFIG=openvpn
      #OPENVPN_OPTS: --inactive 3600 --ping 10 --ping-exit 60
      LOCAL_NETWORK="$LOCAL_NETWORK"
      UMASK_SET=2
      TRANSMISSION_WEB_UI=combustion
      TRANSMISSION_RPC_AUTHENTICATION_REQUIRED="true"
      TRANSMISSION_RPC_HOST_WHITELIST="127.0.0.1,$SERVER_IP"
      TRANSMISSION_RPC_USERNAME=$TRANSMISSION_RPC_USERNAME
      TRANSMISSION_RPC_PASSWORD=$TRANSMISSION_RPC_PASSWORD
      TRANSMISSION_UMASK=002
      TRANSMISSION_RATIO_LIMIT=0.01
      TRANSMISSION_RATIO_LIMIT_ENABLED="true"
      TRANSMISSION_ALT_SPEED_DOWN=40000
      TRANSMISSION_ALT_SPEED_ENABLED="false"
      TRANSMISSION_ALT_SPEED_UP=250
      TRANSMISSION_SPEED_LIMIT_DOWN=100000
      TRANSMISSION_SPEED_LIMIT_DOWN_ENABLED="true"
      TRANSMISSION_SPEED_LIMIT_UP=500
      TRANSMISSION_SPEED_LIMIT_UP_ENABLED="true"
      TRANSMISSION_INCOMPLETE_DIR=/data/torrents/incoming
      TRANSMISSION_INCOMPLETE_DIR_ENABLED="true"
      TRANSMISSION_WATCH_DIR=/data/torrents/watch
      TRANSMISSION_WATCH_DIR_ENABLED="true"
      TRANSMISSION_DOWNLOAD_DIR=/data/downloads/completed
      LOG_TO_STDOUT="true"
    labels:
      - "traefik.enable=true"
      ## HTTP Routers
      - "traefik.http.routers.transmission-vpn-rtr.entrypoints=https"
      - "traefik.http.routers.transmission-vpn-rtr.rule=Host(`transmission.$DOMAINNAME0`)"
      ## Middlewares
      - "traefik.http.routers.transmission-vpn-rtr.middlewares=middlewares-rate-limit@file"
      ## HTTP Services
      - "traefik.http.routers.transmission-vpn-rtr.service=transmission-vpn-svc"
      - "traefik.http.services.transmission-vpn-svc.loadbalancer.server.port=9091"

### Current Behavior

Following the documentation [here](https://haugene.github.io/docker-transmission-openvpn/config-options/#alternative_web_uis), I'm trying to implement another Web-UI but with no success. When I reopen the web page, it's still the official UI.
I've tried all of the custom UI proposed and I've actually no idea where to dig in to find the error. There is no particular information provided in the logs that could help me.

Does someone experienced the same problem?

### Log output

Starting container with revision: 8cc1870cc039201e0e2f8b7684a9f4e96ae03ab9
Creating TUN device /dev/net/tun
mknod: /dev/net/tun: File exists
Using OpenVPN provider: CYBERGHOST
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for CYBERGHOST. Defaulting to external config
Downloading configs from https://github.com/haugene/vpn-configs-contrib/archive/main.zip into /tmp/tmp.Tvm8t713mi
Extracting configs to /tmp/tmp.zIBr1J0tqn
ERROR: Could not find any configs for provider CYBERGHOST in downloaded configs
Cleanup: deleting /tmp/tmp.Tvm8t713mi and /tmp/tmp.zIBr1J0tqn
Starting OpenVPN using config openvpn.ovpn
Modifying /etc/openvpn/cyberghost/openvpn.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
Modification: Change ca certificate path
Modification: Change ping options
Modification: Update/set resolv-retry to 15 seconds
Modification: Change tls-crypt keyfile path
Modification: Set output verbosity to 3
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
Setting OpenVPN credentials...
adding route to local network 192.168.0.0/16 via 192.168.90.1 dev eth0
Wed Mar  9 09:58:29 2022 WARNING: file '/etc/openvpn/cyberghost/client.key' is group or others accessible
Wed Mar  9 09:58:29 2022 OpenVPN 2.4.7 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Jul 19 2021
Wed Mar  9 09:58:29 2022 library versions: OpenSSL 1.1.1f  31 Mar 2020, LZO 2.10
Wed Mar  9 09:58:29 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Wed Mar  9 09:58:29 2022 TCP/UDP: Preserving recently used remote address: [AF_INET]XXX.XXX.XXX.XXX:443
Wed Mar  9 09:58:29 2022 Socket Buffers: R=[212992->212992] S=[212992->212992]
Wed Mar  9 09:58:29 2022 UDP link local: (not bound)
Wed Mar  9 09:58:29 2022 UDP link remote: [AF_INET]102.129.143.38:443
Wed Mar  9 09:58:29 2022 TLS: Initial packet from [AF_INET]XXX.XXX.XXX.XXX:443, sid=16c16592 97fdd9bf
Wed Mar  9 09:58:29 2022 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
Wed Mar  9 09:58:29 2022 VERIFY OK: depth=1, C=RO, L=Bucharest, O=CyberGhost S.A., CN=CyberGhost Root CA, emailAddress=info@cyberghost.ro
Wed Mar  9 09:58:29 2022 VERIFY KU OK
Wed Mar  9 09:58:29 2022 Validating certificate extended key usage
Wed Mar  9 09:58:29 2022 ++ Certificate has EKU (str) TLS Web Server Authentication, expects TLS Web Server Authentication
Wed Mar  9 09:58:29 2022 VERIFY EKU OK
Wed Mar  9 09:58:29 2022 VERIFY OK: depth=0, CN=huenenberg-rack410.nodes.gen4.ninja
Wed Mar  9 09:58:30 2022 WARNING: 'link-mtu' is used inconsistently, local='link-mtu 1569', remote='link-mtu 1549'
Wed Mar  9 09:58:30 2022 WARNING: 'cipher' is used inconsistently, local='cipher AES-256-CBC', remote='cipher AES-128-GCM'
Wed Mar  9 09:58:30 2022 WARNING: 'auth' is used inconsistently, local='auth SHA256', remote='auth [null-digest]'
Wed Mar  9 09:58:30 2022 WARNING: 'keysize' is used inconsistently, local='keysize 256', remote='keysize 128'
Wed Mar  9 09:58:30 2022 Control Channel: TLSv1.3, cipher TLSv1.3 TLS_AES_256_GCM_SHA384, 4096 bit RSA
Wed Mar  9 09:58:30 2022 [huenenberg-rack410.nodes.gen4.ninja] Peer Connection Initiated with [AF_INET]XX.XX.XXX.XXX:443
Wed Mar  9 09:58:31 2022 SENT CONTROL [huenenberg-rack410.nodes.gen4.ninja]: 'PUSH_REQUEST' (status=1)
Wed Mar  9 09:58:31 2022 PUSH: Received control message: 'PUSH_REPLY,redirect-gateway def1,route-ipv6 2000::/3,dhcp-option DNS 10.0.0.243,route-gateway 10.13.4.1,topology subnet,ping 10,ping-restart 60,ifconfig 10.13.4.37 255.255.255.0,peer-id 7'
Wed Mar  9 09:58:31 2022 OPTIONS IMPORT: timers and/or timeouts modified
Wed Mar  9 09:58:31 2022 OPTIONS IMPORT: --ifconfig/up options modified
Wed Mar  9 09:58:31 2022 OPTIONS IMPORT: route options modified
Wed Mar  9 09:58:31 2022 OPTIONS IMPORT: route-related options modified
Wed Mar  9 09:58:31 2022 OPTIONS IMPORT: --ip-win32 and/or --dhcp-option options modified
Wed Mar  9 09:58:31 2022 OPTIONS IMPORT: peer-id set
Wed Mar  9 09:58:31 2022 OPTIONS IMPORT: adjusting link_mtu to 1624
Wed Mar  9 09:58:31 2022 /sbin/ip addr add dev tun0 10.13.4.37/24 broadcast 10.13.4.255
Wed Mar  9 09:58:31 2022 /etc/openvpn/tunnelUp.sh tun0 1500 1572 10.13.4.37 255.255.255.0 init
Up script executed with tun0 1500 1572 10.13.4.37 255.255.255.0 init
Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.13.4.37
Updating Transmission settings.json with values from env variables
Using existing settings.json for Transmission /data/transmission-home/settings.json
Overriding alt-speed-down because TRANSMISSION_ALT_SPEED_DOWN is set to 40000
Overriding alt-speed-enabled because TRANSMISSION_ALT_SPEED_ENABLED is set to false
Overriding alt-speed-up because TRANSMISSION_ALT_SPEED_UP is set to 250
sed'ing True to true
Enforcing ownership on transmission config directories
Applying permissions to transmission config directories
Setting owner for transmission paths to 1027:100
Setting permissions for download and incomplete directories
Mask: 002
Directories: 775
Files: 664
Setting permission for watch directory (775) and its files (664)

Transmission will run as

User name:   abc
User gid:    100

STARTING TRANSMISSION
Transmission startup script complete.
[2022-03-09 09:58:32.726] Transmission 2.94 (d8e60ee44f) started (session.c:740)
[2022-03-09 09:58:32.726] RPC Server Adding address to whitelist: 127.0.0.1 (rpc-server.c:971)
[2022-03-09 09:58:32.727] RPC Server Adding address to whitelist: 192.168.1.2 (rpc-server.c:971)
[2022-03-09 09:58:32.727] RPC Server Adding address to whitelist: 127.0.0.1 (rpc-server.c:971)
[2022-03-09 09:58:32.727] RPC Server Adding address to whitelist: ::1 (rpc-server.c:971)
[2022-03-09 09:58:32.727] RPC Server Serving RPC and Web requests on port 127.0.0.1:9091/transmission/ (rpc-server.c:1213)
[2022-03-09 09:58:32.727] RPC Server Password required (rpc-server.c:1220)
[2022-03-09 09:58:32.727] Port Forwarding Stopped (port-forwarding.c:180)
[2022-03-09 09:58:32.727] DHT Reusing old id (tr-dht.c:307)
[2022-03-09 09:58:32.727] DHT Bootstrapping from 40 IPv4 nodes (tr-dht.c:156)
Wed Mar  9 09:58:36 2022 /sbin/ip route add 102.129.143.38/32 via 192.168.90.1
Wed Mar  9 09:58:36 2022 /sbin/ip route add 0.0.0.0/1 via 10.13.4.1
Wed Mar  9 09:58:36 2022 /sbin/ip route add 128.0.0.0/1 via 10.13.4.1
Wed Mar  9 09:58:36 2022 WARNING: OpenVPN was configured to add an IPv6 route over tun0. However, no IPv6 has been configured for this interface, therefore the route installation may fail or may not work as expected.
Wed Mar  9 09:58:36 2022 Initialization Sequence Completed

### HW/SW Environment

```markdown
- OS: DS920+ Synology
- Docker: docker-compose 3.9
```
.

**Comment by pkishino** [comment]
This is not a bug, sigh.. you need to clear your browser cache

**Reply by thibaudbrg** [reply]
I had already done that but I finally found the solution by removing the docker image and reinstalling the container. Thanks anyway.
